AnnoDesigner/AnnoCanvas.xaml.cs

```diff
@@ -201,6 +201,30 @@ public bool RenderTrueInfluenceRange
             }
         }
 
+        /// <summary>
+        /// Backing field of the SelectAllBuildingsOfType property.
+        /// </summary>
+        private bool _selectAllBuildingsOfType;
+
+        /// <summary>
+        /// Gets or sets a value indicating whether when selecting objects, all the ones with the same identifier should be selected.
+        /// </summary>
+        public bool SelectAllBuildingsOfType
```

Comment:
Maybe a name like `ApplyToObjectsWithSameIdentifier` would be a better fit.

Reply:
I initially meant it to only affect selection/deselection. `ApplyToObjectsWithSameIdentifier` seems to suggest that it would cover everything including deleting via right-click. I can imagine forgetting to turn this setting off, trying to remove a single building and then deleting half of my design.

`SelectObjectsWithSameIdentifier`?

Reply:
Ok, fair point.
Since it is not only selecting, how about `ApplySelectionToObjectsWithSameIdentifier`?
I don't know. A bit long and not catchy in my opinion.